### ISSUE TITLE
Controller numbers must be between 0 -119

### ISSUE THREAD

**Issue by TimSusa** [djipco/webmidi#118]
Hi, 

I just want to kindly ask about this issue. Is this wanted behaviour or is there something wrong on the implementation on my side?

https://github.com/TimSusa/midi-bricks-mono/issues/196

<img width="839" alt="Screenshot 2020-08-01 at 16 19 19" src="https://user-images.githubusercontent.com/8963529/89103693-ff3ee300-d413-11ea-8afc-ed08acee82fb.png">

**Comment by djipco** [comment]
Hello Tim,

As per the [MIDI specification](https://www.midi.org/specifications-old/item/table-3-control-change-messages-data-bytes-2), "Controller numbers 120-127 are reserved for Channel Mode Messages, which rather than controlling sound parameters, affect the channel's operating mode."

This is why WebMidi.js offers a separate [sendChannelMode()](https://webmidijs.org/docs/v2.5.1/classes/Output.html#method_sendChannelMode) method to cover the following messages:

* allsoundoff (#120)
* resetallcontrollers (#121)
* localcontrol (#122)
* allnotesoff (#123)
* omnimodeoff (#124)
* omnimodeon (#125)
* monomodeon (#126)
* polymodeon (#127)

I'd argue that those are quite different from your regular control change message such as modulation wheel or pan. Does that make sense or am I interpreting the specification too narrowly in your opinion?

**Comment by TimSusa** [comment]
Hi, 

as for my feelings specification is specification. So if they say the other numbers are reserved, I am ok with that. The question of one of my users more was, how do the others get this problem to work? Just abusing the standard? I try to do more research on it. Thx for quick answering!

Cheers Tim!

**Comment by djipco** [comment]
> The question of one of my users more was, how do the others get this problem to work?

Do they? I'm curious to know about that as well. Channel mode messages have very well defined functions. If users are able to use them as regular control change messages, it must mean that their devices are ignoring part of the standard... If you find out more, let me know.

Cheers!

**Comment by TimSusa** [comment]
OK, for so far, I would close this issue. Specification is specification!

THX!

**Comment by djipco** [comment]
I took some time today to look at this more carefully. While "_Controller numbers 120-127 are reserved for Channel Mode Messages_", they do are a subset of control change messages. So, in version 3, I decided to trigger **controlchange** events for all controllers (0-127) while also triggering specific channel mode messages for 120-127. 

This agrees with the spec while, at the same time, offering more convenience. This has been integrated in v3.0.0-alpha.11.
